firmware/application/apps/ui_text_editor.hpp

```diff
@@ -75,12 +75,20 @@ class TextViewer : public Widget {
     // Gets the length of the current line.
     uint16_t line_length();
 
-   private:
-    static constexpr int8_t char_width = 5;
-    static constexpr int8_t char_height = 8;
+    void text_zoom(Rect parent_rect) {
+        zoom = !zoom;
+        char_height = zoom ? 16 : 8;
+        char_width = zoom ? 8 : 5;
+        max_line = (uint8_t)(parent_rect.height() / char_height);
+        max_col = (uint8_t)(parent_rect.width() / char_width);
+    }
 
-    const uint8_t max_line = 32;
-    const uint8_t max_col = 48;
+   private:
+    bool zoom = 0;
```

Comment:
`bool` are `false` :)
Not for this PR, but this might be a good setting to put into pmem.
I really need to finish the "generic" app settings support so you can easily stash UI state, but for now, pmem would work.